src/hearings/containers/linked-hearings/linked-hearings-check-your-answers/linked-hearings-check-your-answers.component.ts

```diff
@@ -63,7 +63,7 @@ export class LinkedHearingsCheckYourAnswersComponent implements OnInit, OnDestro
   }
 
   public ngOnInit(): void {
-    this.sub = this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState), take(4)).subscribe(
+    this.sub = this.hearingStore.pipe(select(fromHearingStore.getHearingsFeatureState)).subscribe(
       state => {
```

Comment:
Whilst you are modifying this subscription I believe it should be upgraded to the standard subscription style: https://angular.io/guide/observables i.e. with next, error, complete. This style is now deprecated.

Reply:
This code change is in Angular 7, and when I try next and error it says _cannot find name 'next'_. Might be one to update after merging into Angular 11?

Reply:
Hmmm really? ok... should be able to do: 

`subscribe({
  next: (x: number) => console.log('Observer got a next value: ' + x)
})`

Reply:
It worked now. I missed the curly braces :) Thank you Phil.